<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntelligentMedicalWidget</class>
 <widget class="QWidget" name="IntelligentMedicalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>457</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>479</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Intelligent Medical System</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>260</height>
      </size>
     </property>
     <property name="title">
      <string>Setting</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>22</y>
        <width>114</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Alarm Phone Number:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="alarmPhoneEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>23</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>22</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>51</y>
        <width>151</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Soft Keyboard</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="num6Btn">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="num7Btn">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="num8Btn">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="num9Btn">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="num2Btn">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="num3Btn">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="num4Btn">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="num5Btn">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="num1Btn">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="num0Btn">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="numPlusBtn">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="backBtn">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>&lt;-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="maxLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/pic/max_light.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="minLight">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/pic/min_light.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="BigSpinBoxEdit" name="maxLightValue" native="true">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="BigSpinBoxEdit" name="minLightValue" native="true">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Current phone:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="currentPhoneNumber">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>光照：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="illumDisplay"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>IR Sensor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TwinklingLabel" name="irSensorLabel">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="pixmap">
             <pixmap resource="res.qrc">:/pic/human.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Gas Sensor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TwinklingLabel" name="gasSensorLabel">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="pixmap">
             <pixmap resource="res.qrc">:/pic/gas.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Rain Sensor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TwinklingLabel" name="rainSensorLabel">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="pixmap">
             <pixmap resource="res.qrc">:/pic/rainUmbrella.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Hall Sensor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TwinklingLabel" name="hallSensorLabel">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="pixmap">
             <pixmap resource="res.qrc">:/pic/hall.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TwinklingLabel</class>
   <extends>QLabel</extends>
   <header>TwinklingLabel.h</header>
  </customwidget>
  <customwidget>
   <class>BigSpinBoxEdit</class>
   <extends>QWidget</extends>
   <header>bigspinboxedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>num0Btn</sender>
   <signal>clicked()</signal>
   <receiver>IntelligentMedicalWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num1Btn</sender>
   <signal>clicked()</signal>
   <receiver>IntelligentMedicalWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num2Btn</sender>
   <signal>clicked()</signal>
   <receiver>IntelligentMedicalWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num3Btn</sender>
   <signal>clicked()</signal>
   <receiver>IntelligentMedicalWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num4Btn</sender>
   <signal>clicked()</signal>
   <receiver>IntelligentMedicalWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num5Btn</sender>
   <signal>clicked()</signal>
   <receiver>IntelligentMedicalWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num6Btn</sender>
   <signal>clicked()</signal>
   <receiver>IntelligentMedicalWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num7Btn</sender>
   <signal>clicked()</signal>
   <receiver>IntelligentMedicalWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num8Btn</sender>
   <signal>clicked()</signal>
   <receiver>IntelligentMedicalWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num9Btn</sender>
   <signal>clicked()</signal>
   <receiver>IntelligentMedicalWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numPlusBtn</sender>
   <signal>clicked()</signal>
   <receiver>IntelligentMedicalWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backBtn</sender>
   <signal>clicked()</signal>
   <receiver>IntelligentMedicalWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSoftKeyClicked()</slot>
 </slots>
</ui>
